### PR TITLE
remove hour limit rate from docs

### LINKED ISSUE #1478: Remove hour rate references in docs
docs have references to hourly limites for activations.
There are only minute rate.

### PR DESCRIPTION
**Pull request by csantanapr**
review and feel free to adjust

Fixes #1478

### CODE REVIEW

File: docs/reference.md [hunk 1]
```diff
@@ -447,7 +447,11 @@ The OpenWhisk API supports request-response calls from web clients. OpenWhisk re
 ## System limits
 
 ### Actions
-OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per hour. The following table lists the default limits for actions.
+OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per minute. 
+
+**Note:** This default limits are for the open source distribution, production deployments like BlueMix likely have higher limits. As an operator or developer you can change some of the limits using ansible inventory variables
```
Comment (markusthoemmes) [inline docs/reference.md]: "Bluemix"

File: docs/reference.md [hunk 2]
```diff
@@ -508,12 +512,11 @@ OpenWhisk has a few system limits, including how much memory an action uses and
 
 ### Triggers
 
-Triggers are subject to a firing rate per minute and per hour as documented in the table below.
+Triggers are subject to a firing rate per minute and per minute as documented in the table below.
```
Comment (markusthoemmes) [inline docs/reference.md]: ~~and per minute~~

**Comment by csantanapr** [comment]
@markusthoemmes thanks for the review, I submitted a second commit 4ec150fb880c944463eb3615287a9ec8de6d5674 with the changes.

File: docs/reference.md [hunk 3]
```diff
@@ -447,7 +447,11 @@ The OpenWhisk API supports request-response calls from web clients. OpenWhisk re
 ## System limits
 
 ### Actions
-OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per hour. The following table lists the default limits for actions.
+OpenWhisk has a few system limits, including how much memory an action uses and how many action invocations are allowed per minute. 
+
+**Note:** This default limits are for the open source distribution, production deployments like Bluemix likely have higher limits. As an operator or developer you can change some of the limits using ansible inventory variables
```
Comment (rabbah) [inline docs/reference.md]: in a second pr you added a blurb on how to change these in ansible - shall we link to it from here?

Comment (csantanapr) [inline docs/reference.md]: Yeah, go ahead if you merge both PRs, add the link

**Comment by rabbah** [comment]
replaced by https://github.com/openwhisk/openwhisk/pull/1532

**Pull request closed without merging**